packages/jest/migrations.json

```diff
@@ -215,6 +227,47 @@
           "addToPackageJson": "devDependencies"
         }
       }
+    },
+    "15.8.0": {
+      "version": "15.8.0-beta.0",
+      "packages": {
+        "jest": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "@types/jest": {
+          "version": "~29.4.0",
+          "alwaysAddToPackageJson": false
+        },
+        "expect": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "@jest/globals": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "jest-jasmine2": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "jest-environment-jsdom": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "ts-jest": {
+          "version": "~29.0.5",
+          "alwaysAddToPackageJson": false
+        },
+        "babel-jest": {
+          "version": "~29.4.1",
+          "alwaysAddToPackageJson": false
+        },
+        "jest-preset-angular": {
+          "version": "~13.0.0",
+          "alwaysAddToPackageJson": false
```

Comment:
Might need to talk with @leosvelperez  about how we go about this with Backwards Compat support in Angular. This could be fine, but I'm still not 100% on the new `--interactive` command for migrate

Reply:
talked with Leo and got it setup in the angular migration.json that would work for new interactive migrations